samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -297,12 +301,18 @@
 	  <h:outputText value="" />
 	  <h:outputText value="" />
 	</h:panelGroup>
+        <f:verbatim>&nbsp;</f:verbatim>
```

Comment:
This type of UI layout work should be done with CSS. The icon should probably be sourced from the common Font Awesome set stored in reference.